decompiler/pipeline/controlflowanalysis/variable_name_generation.py

```diff
@@ -71,12 +70,17 @@ def __init__(self, task: DecompilerTask) -> None:
         collector.visit_ast(task._ast)
         self._params: List[Variable] = task._function_parameters
         self._loop_vars : List[Variable] = collector.get_loop_variables()  
-        self._variables: List[Variable] = list(filter(self._filter_variables, collector.get_variables()))  
+        self._variables: List[Variable] = list(filter(self._filter_variables, collector.get_variables()))    
         
 
     def _filter_variables(self, item: Variable) -> bool:
-        """Return False if variable is a parameter, renamed loop variable or GlobalVariable, else True"""
-        if item in self._params or (item in self._loop_vars and item.name.find("var_") == -1) or isinstance(item, GlobalVariable):
+        """Return False if variable is a:
+            - parameter
+            - renamed loop variable
+            - GlobalVariable
+        """
+        if item in self._params or (item in self._loop_vars and item.name.find("var_") == -1) or \
+            isinstance(item, GlobalVariable):
             return False
         return True
```

Comment:
Please invert the logsic of the check, so we can go for  a simple return
so instead of: `if a OR b OR c: return false` go for `return !a AND !b AND !c`

Reply:
Done